packages/fluentui/docs/src/examples/components/Alert/Usage/AlertExampleDismissActions.shorthand.tsx

```diff
@@ -8,8 +8,8 @@ const AlertExampleDismissActions = () => {
   return (
     <Alert
       actions={[
-        { content: 'Privacy policy', primary: true },
-        { content: 'Dismiss', onClick: () => setVisible(false) },
+        { content: 'Privacy policy', primary: true, key: 'privacy' },
```

Comment:
look here :D `privacy` is the same as in the example above. It might cause some strange behavior.

https://egghead.io/lessons/react-use-the-key-prop-when-rendering-a-list-with-react-12564a86

Reply:
It makes no diff since are separated list. Does it?